crates/daphne/src/taskprov.rs

```diff
@@ -63,16 +63,14 @@ pub(crate) fn extract_prk_from_verify_key_init(
     Salt::new(HKDF_SHA256, &TASKPROV_SALT).extract(verify_key_init)
 }
 
-impl VdafConfig {
-    fn expand_into_taskprov_verify_key(&self, prk: &Prk, task_id: &TaskId) -> VdafVerifyKey {
-        let mut verify_key = self.uninitialized_verify_key();
-        let info = [task_id.as_ref()];
-        // This expand(), and the associated fill() below can only fail if the length is wrong,
-        // and it won't be, so we unwrap().
-        let okm = prk.expand(&info, verify_key.clone()).unwrap();
-        okm.fill(verify_key.as_mut()).unwrap();
-        verify_key
-    }
+fn expand_into_taskprov_verify_key(prk: &Prk, task_id: &TaskId) -> VdafVerifyKey {
+    let mut verify_key = VdafVerifyKey::uninitialized();
+    let info = [task_id.as_ref()];
+    // This expand(), and the associated fill() below can only fail if the length is wrong,
+    // and it won't be, so we unwrap().
+    let okm = prk.expand(&info, verify_key.clone()).unwrap();
+    okm.fill(verify_key.as_mut()).unwrap();
+    verify_key
 }
```

Comment:
Instead of having an `uninitialized` function you could instead `impl TryFrom<&[u8]> for VdafVerifyKey` and use that to create it already initialized